plugins/services/src/js/components/ConfigurationMapTable.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {Table} from 'reactjs-components';
+
+import ValidatorUtil from '../../../../../src/js/utils/ValidatorUtil.js';
+
+/**
+ * Optimised method to check if all row props are empty for a given column
+ *
+ * @param {Array} data - The array of rows top rocess
+ * @param {String} prop - The property name of the column
+ * @returns {Boolean} Returns `true` if all rows have empty value in this prop
+ */
+function isColumnEmpty(data, prop) {
+  for (let i=0, length=data.length; i<length; ++i) {
+    if (!ValidatorUtil.isEmpty(data[i][prop])) {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+/**
+ * Custom rendering function that takes care of replacing with default value
+ * if the field is empty.
+ *
```

Comment:
Let's add `@this` section and a bit of a description that we're going to bind the function